### PR TITLE
Restore `default-branch-button`

### PR DESCRIPTION
**Pull request by fregante**
- Part of #6152 

## Test URLs

Branch, subfolder: https://github.com/refined-github/refined-github/tree/better-ajax-detection/safari

Screenshot 14

Commit, home page: https://github.com/refined-github/refined-github/tree/2ce3317eba7d01d0c728605ad7730b4ee067835d

Screenshot 13

### CODE REVIEW

**Comment by fregante** [comment]
Also untouched on the main branch, both on the home and in a subfolder. They have completely different markup

Screenshot 15

Screenshot 16

**Comment by fregante** [comment]
Note that in some views, GitHub may be display the same button twice. I'm not gonna address this now as I think they might _fix it_ eventually. 

Screenshot 22



Anyway the feature is applied to both places 🤷‍♂️ 

Screenshot 21

File: source/refined-github.css [hunk 1]
```diff
@@ -191,3 +191,9 @@ div[data-target='readme-toc.content'] div.blob-wrapper img {
 [data-selected-links^='repo_milestones ']:has([title='0']) svg {
 	margin-right: 20px; /* Push the text farther away from the text so it's cropped out */
 }
+
+/* Add override for CSS-in-JSS components that don't use regular .btn styles */
```
Comment (FloEdelmann) [inline source/refined-github.css:195]:
```suggestion
/* Add override for CSS-in-JS components that don't use regular .btn styles */
```

Comment (fregante) [reply source/refined-github.css:195]:
```suggestion
/* Add override for CSS-in-JSSS components that don't use regular .btn styles */
```

https://en.wikipedia.org/wiki/JavaScript_Style_Sheets

Comment (FloEdelmann) [reply source/refined-github.css:195]:
```suggestion
/* Add override for CS-in-JSS components that don't use regular .btn styles */
```

https://en.wikipedia.org/wiki/Computer_science

Comment (fregante) [reply source/refined-github.css:195]:
```suggestion
/* Add override for JAXA-in-ISS components that don't use regular .btn styles */
```

https://en.wikipedia.org/wiki/International_Space_Station

**Pull request merged**